pkg/api.go

```diff
@@ -100,6 +101,21 @@ type S3Check struct {
 	ObjectPath string   `yaml:"objectPath"`
 }
 
+type S3BucketCheck struct {
+	Bucket    string `yaml:"bucket"`
```

Comment:
Can we make this support env variable like: https://github.com/flanksource/canary-checker/blob/master/pkg/config.go#L30